modules/cudl-data-processing/lambda.tf

```diff
@@ -68,48 +68,48 @@ resource "aws_lambda_alias" "create-transform-lambda-alias" {
   depends_on = [aws_lambda_function.create-transform-lambda-function]
 }
 
-resource "aws_lambda_function" "create-db-lambda-function" {
```

Comment:
Commenting here as unable to comment on the resources `aws_lambda_layer_version.transform-properties-layer` and `aws_lambda_layer_version.db-properties-layer`. Given the resource `aws_lambda_function.create-db-lambda-function` is being removed, would it make sense to combine the two `aws_lambda_layer_version` resources into one object, as they essentially have the same data. Also they have a reference to the variable `db-lambda-information` which could be removed

Reply:
I have removed the `aws_lambda_layer_version.db-properties-layer `layer as it seemed to contain the same information and is included now using the` transform-properties-layer`